src/etc/inc/services.inc

```diff
@@ -249,51 +249,53 @@ function services_radvd_configure($blacklist = array()) {
 		$radvdconf .= "\t};\n";
 
 		/* add DNS servers */
-		$dnslist = array();
-		if (isset($dhcpv6ifconf['rasamednsasdhcp6']) && is_array($dhcpv6ifconf['dnsserver']) && !empty($dhcpv6ifconf['dnsserver'])) {
-			foreach ($dhcpv6ifconf['dnsserver'] as $server) {
-				if (is_ipaddrv6($server)) {
-					$dnslist[] = $server;
+		if (($dhcpv6ifconf['enable-radvd-dns']) != 'disabled') {
```

Comment:
Calling this `enable-radvd-dns` when it will contain a value of `enabled` or `disabled` seems a bit redundant, why not just `radvd-dns`?